conf/locale/locale_de-DE.ini

```diff
@@ -815,6 +815,7 @@ team_permission_desc=Welche Berechtigungsstufe soll das Team haben?
 
 form.name_reserved=Organisationsname '%s' ist bereits vergeben.
 form.name_pattern_not_allowed=Organisationsnamen der Form '%s' sind nicht erlaubt.
+form.create_org_not_allowed=Dieser Benutzer darf keine Organisation erstellen.
```

Comment:
I think you're supposed to only modify the english localization, whereas other languages are delegated to an external service. Please @lunny, @bkcsoft, @tboerger  correct me if I'm wrong